lib/models/bundle.js

```diff
@@ -262,29 +264,40 @@ Bundle.create = function (opts, callback) {
     // Prevent Zalgo by executing on the next tick.
     return setImmediate(() => {
       const bundle = new Bundle({
-        code: opts.code
+        code: opts.code,
+        allowEmpty: opts.allowEmpty
       });
 
-      bundle.validate();
+      let err = null;
+      try {
+        bundle.validate();
+      } catch (validateErr) {
+        err = validateErr;
+      }
 
-      callback(null, bundle);
+      callback(err, bundle);
     });
   }
 
-  return fs.readFile(opts.bundle, (err, data) => {
-    if (err) {
-      callback(err);
-      return;
+  return fs.readFile(opts.bundle, (readErr, data) => {
+    if (readErr) {
+      return void callback(readErr);
```

Comment:
It's how you do an eslint blessed inconsistent return for this specification: https://github.com/FormidableLabs/inspectpack/blob/master/.eslintrc#L12

```js
consistent-return: [error, { treatUndefinedAsUnspecified: true }]
```

See: http://eslint.org/docs/rules/consistent-return#treatundefinedasunspecified

Probably worth updating upstream `eslint-config-formidable` if other folks like it. (I usually have this specification in my eslintrc's)